table.go

```diff
@@ -47,31 +57,133 @@ type RoutingTable struct {
 	maxLatency time.Duration
 
 	// kBuckets define all the fingers to other nodes.
-	Buckets    []*Bucket
+	Buckets    []*bucket
```

Comment:
If we're no longer exporting the bucket type, we shouldn't export this field.

Reply:
I've made the change.